.github/workflows/pre-merge-checks.yml

```diff
@@ -98,6 +98,60 @@ jobs:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         run: ./gradlew sonar
 
+  api-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    permissions:
+      id-token: write
+      packages: read
+      contents: read
+    defaults:
+      run:
+        working-directory: ./api-tests
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
```

Comment:
I think that the api-tests were written with node 20

Reply:
Ah, I must have blindly copied this from delete-user-data (which should also use node 20...)